.ddev/config.yaml

```diff
@@ -0,0 +1,267 @@
+type: drupal
+docroot: web
+php_version: "8.3"
+webserver_type: nginx-fpm
+router_http_port: "80"
+router_https_port: "443"
+xdebug_enabled: false
+additional_hostnames:
+ - drupal-core-testing
+additional_fqdns: []
+database:
+  type: mariadb
+  version: "10.11"
+nfs_mount_enabled: false
+mutagen_enabled: false
```

Comment:
This is an obsolete configuration. We now use `performance_mode` and have for some time. But it is not normally useful at the project level.
```suggestion
```